i18n-util/webpack.config.js

```diff
@@ -6,7 +6,10 @@ module.exports = {
   entry: './src/main.js',
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename: 'i18n-util.bundle.[fullhash].js'
+    filename: 'i18n-util.bundle.[fullhash].js',
+    environment: {
+      arrowFunction: false
```

Comment:
This was needed because webpack was injecting an arrow function **after** babel-loader ran, still breaking IE11.

See https://github.com/babel/babel-loader#top-level-function-iife-is-still-arrow-on-webpack-5